js/script.js

```diff
@@ -1,109 +1,66 @@
+Book.displayBooks();
+const listLink = document.querySelector('#list-link');
+const newLink = document.querySelector('#new-link');
+const contactLink = document.querySelector('#contact-link');
 
-  const listSection = document.querySelector('.books-list');
-  const newSection = document.querySelector('.add-books');
-  const contactSection = document.querySelector('.contact');
+const listSection = document.querySelector('.books-list');
+const newSection = document.querySelector('.add-books');
+const contactSection = document.querySelector('.contact');
 
-  listLink.addEventListener('click', () => {
-    listSection.classList.add('show');
-    newSection.classList.remove('show');
-    contactSection.classList.remove('show');
-    listLink.classList.add('active');
-    newLink.classList.remove('active');
-    contactLink.classList.remove('active)');
-  });
+listLink.addEventListener('click', () => {
+  listSection.classList.add('show');
+  newSection.classList.remove('show');
+  contactSection.classList.remove('show');
+  listLink.classList.add('active');
+  newLink.classList.remove('active');
+  contactLink.classList.remove('active)');
+});
 
-  newLink.addEventListener('click', () => {
-    listSection.classList.remove('show');
-    newSection.classList.add('show');
-    contactSection.classList.remove('show');
-    listLink.classList.remove('active');
-    newLink.classList.add('active');
-    contactLink.classList.remove('active)');
-  });
+newLink.addEventListener('click', () => {
+  listSection.classList.remove('show');
+  newSection.classList.add('show');
+  contactSection.classList.remove('show');
+  listLink.classList.remove('active');
+  newLink.classList.add('active');
+  contactLink.classList.remove('active)');
+});
 
-  contactLink.addEventListener('click', () => {
-    listSection.classList.remove('show');
-    newSection.classList.remove('show');
-    contactSection.classList.add('show');
-    listLink.classList.remove('active');
-    newLink.classList.remove('active');
-    contactLink.classList.add('active');
-  });
+contactLink.addEventListener('click', () => {
+  listSection.classList.remove('show');
+  newSection.classList.remove('show');
+  contactSection.classList.add('show');
+  listLink.classList.remove('active');
+  newLink.classList.remove('active');
+  contactLink.classList.add('active');
+});
 
-  const time = document.querySelector('.time');
-  const now = luxon.DateTime.now();// eslint-disable-line no-undef
-  const date = luxon.DateTime.fromISO(now);// eslint-disable-line no-undef
-  time.innerHTML = date.toLocaleString(luxon.DateTime.DATETIME_MED);// eslint-disable-line no-undef
-};
\ No newline at end of file
+const time = document.querySelector('.time');
+const now = luxon.DateTime.now();// eslint-disable-line no-undef
+const date = luxon.DateTime.fromISO(now);// eslint-disable-line no-undef
+time.innerHTML = date.toLocaleString(luxon.DateTime.DATETIME_MED);// eslint-disable-line no-undef
```

Comment:
- [ ] Kindly follow all the [steps to install `luxon`](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md). The requirement says that you should have a `packaje.json` fine in your project this fie is the one that will hold the `luxon dependency. 
- [ ] Also instead of disabling linters, kindly try to solve them.